google/cloud/alloydb/connector/connector.py

```diff
@@ -51,9 +54,10 @@ def __init__(
         credentials: Optional[Credentials] = None,
         quota_project: Optional[str] = None,
         alloydb_api_endpoint: str = "https://alloydb.googleapis.com",
+        loop: asyncio.AbstractEventLoop = None,
     ) -> None:
         # create event loop and start it in background thread
-        self._loop: asyncio.AbstractEventLoop = asyncio.new_event_loop()
+        self._loop: asyncio.AbstractEventLoop = loop or asyncio.new_event_loop()
```

Comment:
The `or` operator is often under debate whether it should be used in this manner for assigning a variable. It is sometime considered an anti-pattern to do this in other languages. I tend to avoid using the `or` for assignment and instead use:
```suggestion
        self._loop: asyncio.AbstractEventLoop = asyncio.new_event_loop() if loop is None else loop
```